.github/workflows/docs.yml

```diff
@@ -0,0 +1,50 @@
+name: Docs
+
+on:
+  push:
+    paths:
+      - ".github/workflows/docs.yml"
+      - "docs/**"
+  pull_request:
+    paths:
+      - ".github/workflows/docs.yml"
```

Comment:
Ok, so you're also running the Docs workflow if the GitHub Actions file changes, fair enough. In that case, shouldn't the same logic also apply to the Valgrind workflow? https://github.com/hugovk/Pillow/pull/95

Reply:
Good idea, merged!